codalab/lib/interactive_session.py

```diff
@@ -80,10 +79,15 @@ def __init__(
         self._stderr = stderr
 
     def start(self):
-        self._bundle_path = os.path.join(
+        self._host_bundle_path = os.path.join(
             self._manager.codalab_home, 'local_bundles', self._session_uuid
         )
-        os.makedirs(self._bundle_path)
+        os.makedirs(self._host_bundle_path)
+
+        # Create a blank file which will be used as the bash history file that will later be
+        # mounted and populated during the interactive session.
```

Comment:
What's the purpose of mounting `.bash_history`?

Reply:
We mount `.bash_history` so we have access to the file on the host after the container exits.